src/EnergyPlus/UtilityRoutines.hh

```diff
@@ -77,14 +77,20 @@ void CloseOutOpenFiles();
 
 int EndEnergyPlus(EnergyPlusData &state);
 
-void ConvertCaseToUpper(std::string const &InputString, // Input string
-                        std::string &OutputString       // Output string (in UpperCase)
+void ConvertCaseToUpper(std::string_view InputString, // Input string
+                        std::string &OutputString     // Output string (in UpperCase)
 );
 
-void ConvertCaseToLower(std::string const &InputString, // Input string
-                        std::string &OutputString       // Output string (in LowerCase)
+void ConvertCaseToLower(std::string_view InputString, // Input string
+                        std::string &OutputString     // Output string (in LowerCase)
 );
 
+// useful for forcing a conversion to a string reference for JSON objects
+inline const std::string &AsString(const std::string &value)
+{
+    return value;
+}
```

Comment:
OK, here's the worker function.  It makes sense as a standalone thing, but I'm having trouble piecing together how it fits with the JSON library tonight.

Reply:
All that's doing is forcing a conversion from a JSON object to std::string.

The "problem" is that many functions now take a `string_view` and if you pass a JSON object to a function requiring a `string_view` then it doesn't compile.

This would be no different then:

`someFunctionThatTakesStringView(static_cast<std::string>(jsonObject))`

I made this helper function as an attempt to make that more clear... but maybe it should instead take a JSON value instead of an `std::string`